### DISCUSSION TITLE
Release Version 1.0.1 to Grafana Cloud

### DISCUSSION THREAD

**Discussion by ignaciood** [grafana/grafana-infinity-datasource#440] [Q&A] [answered]
Hi how are you?
My question is if this version (1.0.1) will be pushed to Grafana Cloud?
Today the version that is available in [Grafana Cloud Plugin](https://grafana.com/grafana/plugins/yesoreyeram-infinity-datasource/?tab=installation) is 1.0.0

Thank you very much! :)

**Comment by yesoreyeram** [comment, marked as answer]
Yes. It will be.. in next couple of weeks. I am testing bunch of stuff before release.

**Reply by ignaciood** [reply]
Thanks for the prompt response! :)
